pyrefinebio/http.py

```diff
@@ -45,6 +53,7 @@ def request(method, url, params=None, payload=None):
             raise ServerError()
 
         else:
+            print(response_body)
```

Comment:
Was this a debugging statement?

Reply:
Oh no I see now, it's to explain the exception in case we don't recognize any other errors.

In that case might I suggest adding an exception for 403 which tells them they need to provided an activated token? If you want to handle this in another PR just make an issue to keep track of it.

Reply:
I guess the API returns a 400 instead of a 403.  Looks like we handle it appropriately then. Nice work!